dark/views/platform/add.py

```diff
@@ -20,8 +21,14 @@ class AddPlatformView(CreateView):
     def form_valid(self, form):
         form.instance.creator = self.request.user
         platform_name = form.cleaned_data.get('name')
-        messages.success(self.request, 'Platform ' + platform_name + ' has successfully added!')
-        return super(AddPlatformView, self).form_valid(form)
+
+        try:
+            messages.success(self.request, 'Platform ' + platform_name + ' has successfully added!')
+            return super(AddPlatformView, self).form_valid(form)
+        except IntegrityError as e:
+            messages.error(self.request, f'Failed to add platform - make sure that field values are valid')
+            return super(AddPlatformView, self).form_invalid(form)
```

Comment:
Nie wiem czy da się to poprawić w sposób łatwy, ale mam coś takiego:
![image](https://user-images.githubusercontent.com/44002074/121253172-1575c500-c8a9-11eb-8e65-5b05b83c55f1.png)
![image](https://user-images.githubusercontent.com/44002074/121253203-1eff2d00-c8a9-11eb-826f-90334e2dc2b3.png)